### DISCUSSION TITLE
blog/go-new-coverage

### DISCUSSION THREAD

**Discussion by giscus** [CarlJi/website#5] [General]
# blog/go-new-coverage

玩过 Go 覆盖率的同学当有所了解，Go 的覆盖率方案最初的设计目标仅是针对单测场景，导致其局限性很大。而为了适配更多的场景，行业内各种博客、插件、黑科技介绍也层出不穷。当然，过去我们也开源过 Go 系统测试覆盖率收集利器 - goc，算其中比较完善，比较系统的了。且从用户使用角度来看，goc 也确实解决了行业内很多同学的痛点。

https://carlji.netlify.app/blog/go-new-coverage

**Comment by CarlJi** [comment]
测试Comment
